{ 'cebit_hands': { 'head': [ { 'duration': 1.64,
                               'name': u'cebit_right_down',
                               'positions': [-0.3768, 0.3817],
                               'starttime': 2.298},
                             { 'duration': 1.512,
                               'name': u'cebit_left_down',
                               'positions': [0.3768, 0.3847],
                               'starttime': 4.438},
                             { 'duration': 1.0,
                               'name': u'cebit_down',
                               'positions': [-0.0, 0.3849],
                               'starttime': 6.462},
                             { 'duration': 0.963,
                               'name': u'ready_state',
                               'positions': [-0.0, 0.0022],
                               'starttime': 7.738},
                             { 'duration': 1.507,
                               'name': u'cebit_down',
                               'positions': [-0.0, 0.3849],
                               'starttime': 9.715},
                             { 'duration': 0.404,
                               'name': u'cebit_right_down',
                               'positions': [-0.3768, 0.3817],
                               'starttime': 11.593},
                             { 'duration': 1.429,
                               'name': u'cebit_head_right',
                               'positions': [-1.1307, 0.0126],
                               'starttime': 11.997},
                             { 'duration': 0.532,
                               'name': u'cebit_left_down',
                               'positions': [0.3768, 0.3847],
                               'starttime': 13.48},
                             { 'duration': 1.468,
                               'name': u'cebit_head_left',
                               'positions': [1.1296, 0.0127],
                               'starttime': 14.012},
                             { 'duration': 1.0,
                               'name': u'ready_state',
                               'positions': [-0.0, 0.0022],
                               'starttime': 16.019}],
                   'left_arm': [ { 'duration': 1.512,
                                   'name': u'cebit_hand_up',
                                   'positions': [ -0.9018,
                                                  -0.4709,
                                                  -0.4977,
                                                  0.7832,
                                                  -0.5479,
                                                  0.0586,
                                                  -0.0001],
                                   'starttime': 4.47},
                                 { 'duration': 1.0,
                                   'name': u'cebit_hands_open',
                                   'positions': [ -1.005,
                                                  -0.1341,
                                                  -0.141,
                                                  0.1254,
                                                  -0.5324,
                                                  -0.0505,
                                                  0.0],
                                   'starttime': 6.388},
                                 { 'duration': 0.68,
                                   'name': u'cebit_hands_open_1',
                                   'positions': [ -1.2754,
                                                  0.2513,
                                                  -0.1258,
                                                  0.1301,
                                                  -0.9424,
                                                  -0.06,
                                                  -0.0],
                                   'starttime': 7.735},
                                 { 'duration': 0.616,
                                   'name': u'cebit_hands_open',
                                   'positions': [ -1.005,
                                                  -0.1341,
                                                  -0.141,
                                                  0.1254,
                                                  -0.5324,
                                                  -0.0505,
                                                  0.0],
                                   'starttime': 8.433},
                                 { 'duration': 1.585,
                                   'name': u'cebit_hands_open_2',
                                   'positions': [ -1.69,
                                                  -0.61,
                                                  -0.9426,
                                                  0.6901,
                                                  -0.6032,
                                                  1.0053,
                                                  -0.0],
                                   'starttime': 9.601},
                                 { 'duration': 1.468,
                                   'name': u'cebit_arm_up',
                                   'positions': [ -1.5492,
                                                  -1.8573,
                                                  -0.7556,
                                                  0.8048,
                                                  -0.3019,
                                                  -0.1253,
                                                  -0.0],
                                   'starttime': 13.989},
                                 { 'duration': 2.024,
                                   'name': u'cebit_wide',
                                   'positions': [ -0.9989,
                                                  -1.6965,
                                                  -1.445,
                                                  0.7999,
                                                  -0.6408,
                                                  -0.13,
                                                  -0.0],
                                   'starttime': 16.977},
                                 { 'duration': 2.024,
                                   'name': u'cebit_wide_2',
                                   'positions': [ -0.999,
                                                  -0.691,
                                                  -1.0054,
                                                  1.3823,
                                                  -0.7162,
                                                  -0.13,
                                                  -0.0],
                                   'starttime': 19.001},
                                 { 'duration': 1.967,
                                   'name': u'ready_state',
                                   'positions': [ -1.5135,
                                                  0.7532,
                                                  -0.2755,
                                                  0.0212,
                                                  0.0208,
                                                  -0.2499,
                                                  0.8168],
                                   'starttime': 21.025000000000002}],
                   'left_leg': [ { 'duration': 1.96,
                                   'name': u'cebit_upright',
                                   'positions': [ 0.0199,
                                                  -0.0901,
                                                  -0.6282,
                                                  1.2101,
                                                  0.6201,
                                                  -0.0899],
                                   'starttime': 0.0},
                                 { 'duration': 2.008,
                                   'name': u'ready_state',
                                   'positions': [ 0.0113,
                                                  -0.0746,
                                                  -0.7899,
                                                  1.1893,
                                                  0.6074,
                                                  -0.0704],
                                   'starttime': 23.002}],
                   'right_arm': [ { 'duration': 1.576,
                                    'name': u'cebit_hand_up',
                                    'positions': [ 0.9018,
                                                   0.4709,
                                                   0.4977,
                                                   -0.7832,
                                                   0.5479,
                                                   -0.0586,
                                                   0.0001],
                                    'starttime': 2.393},
                                  { 'duration': 1.0,
                                    'name': u'cebit_hands_open',
                                    'positions': [ 1.005,
                                                   0.1341,
                                                   0.141,
                                                   -0.1254,
                                                   0.5324,
                                                   0.0505,
                                                   -0.0],
                                    'starttime': 6.388},
                                  { 'duration': 0.68,
                                    'name': u'cebit_hands_open_1',
                                    'positions': [ 1.2754,
                                                   -0.2513,
                                                   0.1258,
                                                   -0.1301,
                                                   0.9424,
                                                   0.06,
                                                   0.0],
                                    'starttime': 7.736},
                                  { 'duration': 0.616,
                                    'name': u'cebit_hands_open',
                                    'positions': [ 1.005,
                                                   0.1341,
                                                   0.141,
                                                   -0.1254,
                                                   0.5324,
                                                   0.0505,
                                                   -0.0],
                                    'starttime': 8.417},
                                  { 'duration': 1.585,
                                    'name': u'cebit_hands_open_2',
                                    'positions': [ 1.69,
                                                   0.61,
                                                   0.9426,
                                                   -0.6901,
                                                   0.6032,
                                                   -1.0053,
                                                   0.0],
                                    'starttime': 9.61},
                                  { 'duration': 1.429,
                                    'name': u'cebit_arm_up',
                                    'positions': [ 1.5492,
                                                   1.8573,
                                                   0.7556,
                                                   -0.8048,
                                                   0.3019,
                                                   0.1253,
                                                   0.0],
                                    'starttime': 12.003},
                                  { 'duration': 2.024,
                                    'name': u'cebit_wide',
                                    'positions': [ 0.9989,
                                                   1.6965,
                                                   1.445,
                                                   -0.7999,
                                                   0.6408,
                                                   0.13,
                                                   0.0],
                                    'starttime': 16.993},
                                  { 'duration': 1.96,
                                    'name': u'cebit_wide_2',
                                    'positions': [ 0.999,
                                                   0.691,
                                                   1.0054,
                                                   -1.3823,
                                                   0.7162,
                                                   0.13,
                                                   0.0],
                                    'starttime': 19.038},
                                  { 'duration': 1.97,
                                    'name': u'ready_state',
                                    'positions': [ 1.5135,
                                                   -0.7532,
                                                   0.2755,
                                                   -0.0212,
                                                   -0.0208,
                                                   0.2499,
                                                   -0.8168],
                                    'starttime': 20.998}],
                   'right_leg': [ { 'duration': 1.96,
                                    'name': u'cebit_upright',
                                    'positions': [ -0.0199,
                                                   0.0901,
                                                   0.6282,
                                                   -1.2101,
                                                   -0.6201,
                                                   0.0899],
                                    'starttime': 0.0},
                                  { 'duration': 2.008,
                                    'name': u'ready_state',
                                    'positions': [ -0.0113,
                                                   0.0746,
                                                   0.7899,
                                                   -1.1893,
                                                   -0.6074,
                                                   0.0704],
                                    'starttime': 23.002}],
                   'torso': [ { 'duration': 1.507,
                                'name': u'cebit_right',
                                'positions': [-0.1569, 0.0103],
                                'starttime': 12.009},
                              { 'duration': 1.468,
                                'name': u'cebit_left',
                                'positions': [0.1093, 0.0105],
                                'starttime': 14.0},
                              { 'duration': 1.0,
                                'name': u'ready_state',
                                'positions': [0.0001, -0.0002],
                                'starttime': 16.003}]}}
